### DISCUSSION TITLE
Use AssetPack plugin on a server without sass installed

### DISCUSSION THREAD

**Discussion by robrwo** [mojolicious/mojo#1847] [Q&A]
Is it possible to use the AssetPack plugin and build Sass (.scss) files into appropriate .css files on a development server, update the caches in a git repo and then run the Mojolicious application on a server without Sass?

In other words, the server should not try and rebuild the CSS files. It should just trust what's in the cache.

**Comment by robrwo** [comment]
I've worked around this by manually building CSS separately.

Is there a way to separate AssetPack's processing that "builds" the files vs one that simply uses the built files, so that a separate "build assets" command can be run by developers, versus the application itself parsing assetpack.db and simply using the existing files?

**Reply by jhthorsen** [reply]
Maybe you want to try out https://metacpan.org/pod/Mojolicious::Plugin::Webpack

I wrote both of the plugins, and I know that the Webpack-version will completely trust the cache, and not try to rebuild anything.
